vm/vm.go

```diff
@@ -0,0 +1,121 @@
+package vm
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"net"
+	"os"
+
+	"github.com/kubearmor/kubearmor-client/k8s"
+	pb "github.com/kubearmor/kubearmor-client/protobuf"
+	"github.com/rs/zerolog/log"
+	"google.golang.org/grpc"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+type VmOptions struct {
+	IP     string
+	VMName string
+	File   string
+}
+
+var (
+	serviceAccountName = "kvmsoperator"
+	pbClient           pb.HandleCliClient
+)
+
+func initGrpcClient(ip string) error {
+	// Connect to gRPC server
+	grpcClientConn, err := grpc.DialContext(context.Background(), net.JoinHostPort(ip, "32770"), grpc.WithInsecure())
+	if err != nil {
+		return err
+	}
+
+	pbClient = pb.NewHandleCliClient(grpcClientConn)
+	if pbClient == nil {
+		return errors.New("invalid grpc client handle")
+	}
+	return nil
+}
+
+func writeScriptDataToFile(options VmOptions, scriptData string) error {
+
+	var filename string
+
+	if options.File == "none" {
+		filename = options.VMName + ".sh"
+	} else {
+		filename = options.File
+	}
+
+	file, err := os.Create(filename)
+	if err != nil {
+		return err
+	}
+	_, err = file.WriteString(scriptData)
+	if err != nil {
+		return err
+	}
+
+	fmt.Printf("VM installation script copied to %s\n", filename)
```

Comment:
better to use log in place of fmt.

Reply:
Other go files use fmt. Should i still carry out with fmt or log ?

Reply:
This golang file uses both `log` and `fmt`. No point in using both. Lets just use `log`. If the same problem exists elsewhere, lets fix that too.